### PR TITLE
SOC-1282 increase timeouts

### PR DESCRIPTION
**Pull request by jsutterfield**
Ticket: https://wikia-inc.atlassian.net/browse/SOC-1282

### CODE REVIEW

File: service/user-attribute/src/main/resources/user-attribute.yaml [hunk 1]
```diff
@@ -36,8 +36,8 @@ swaggerConfiguration:
     authKey: ${consul:SWAGGER_AUTH_HEADER:X-Wikia-UserId}
     authKeyLocation: ${consul:SWAGGER_SECURITY_TOKEN_TYPE:header}
 httpClientConfiguration:
-  timeout: 500ms
-  connectionTimeout: 500ms
+  timeout: 2s
+  connectionTimeout: 2s
```
Comment (jsutterfield) [inline service/user-attribute/src/main/resources/user-attribute.yaml]: I'll be honest, I'm not sure what kind of science to apply here. 2 seconds seems like a reasonable amount of time. We had it set at 500ms before and only 1/10th of 1% of requests had exceptions, so very few requests will be affected by this timeout change.

If anyone has any strong opinions about why this should be higher or lower, or knows best practices when it comes to defining this kind of timeout, I'm happy to hear it.

Comment (armonr) [inline service/user-attribute/src/main/resources/user-attribute.yaml]: I think @macbre's suggestion (in the ticket) is the preferable solution.
As a stopgap measure, though, this LGTM; also `retries` might help as long as the timeouts are not happening during traffic surges when, for instance, connection limit is reached.

Comment (jsutterfield) [inline service/user-attribute/src/main/resources/user-attribute.yaml]: Agreed, using the purger queue would be prefereable, but right now there's no way to talk to it. See [this comment](https://wikia-inc.atlassian.net/browse/SOC-1282?focusedCommentId=196882&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-196882)

**Comment by garthwebb** [comment]
LGTM

**Pull request merged**